### PR TITLE
SO-4826: FHIR ValueSet/ConceptMap operations

### PR DESCRIPTION
**Pull request by cmark**
Resurrect ValueSet and ConceptMap operations using the new 8.x feature and improvements.

### CODE REVIEW

File: fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirConceptMapTranslateOperationController.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		@Parameter(description = "If true, the mapping is reversed") @RequestParam(value="reverse") final Optional<Boolean> isReverse) {
+		
+		//validation is triggered by builder.build()
+		Builder builder = TranslateRequest.builder()
+			.code(code)
+			.system(system);
+		
+		if (version.isPresent()) {
+			builder.version(version.get());
+		}
```
Comment (apeteri) [inline fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirConceptMapTranslateOperationController.java]: `Optional` offers a shortcut for these blocks, it should also work with a method reference as the consumer:

```suggestion
		version.ifPresent(builder::version);
```

File: fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirValueSetExpandOperationController.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+		}
+
+		if (request.getUrl() == null || request.getUrl().getUriValue() == null) {
+			throw new BadRequestException("Expand request URL is not defined.", "ExpandValueSetRequest");
+		}
+		
+		if (request.getUrl() != null && 
+				request.getValueSet() != null && 
+				request.getUrl().getUriValue() != null &&
+				request.getValueSet().getUrl().getUriValue() != null &&
```
Comment (apeteri) [inline fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirValueSetExpandOperationController.java:124]: We don't know whether `request.getValueSet().getUrl()` is non-null at this point 😃

**Review by apeteri — APPROVED** [review]

File: fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirConceptMapTranslateOperationController.java (hunk 1 above)
Comment (cmark) [reply fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirConceptMapTranslateOperationController.java]: Yeah, I have just copy-pasted the original code 😅, but will apply these improvements.

File: fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirValueSetExpandOperationController.java (hunk 2 above)
Comment (cmark) [reply fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirValueSetExpandOperationController.java:124]: Same here, this was the original code in 7.x. Also, we won't support the `valueSet` property of the `$expand` operation, feel free to ignore this for now.

**Pull request merged**